rotkehlchen/globaldb/handler.py

```diff
@@ -84,6 +84,16 @@
 )
 
 
+def _prioritize_manual_balances_query() -> tuple[str, str]:
+    """Prioritize manual price if it exists and choose the timestamp closest to the target.
+    Return the order statement for the query and the type to prioritize already serialized.
+    """
+    return (
+        ' ORDER BY CASE WHEN source_type=? THEN 0 ELSE 1 END, ABS(timestamp - ?)',
+        HistoricalPriceOracle.MANUAL.serialize_for_db(),
```

Comment:
Can you:

1. Change this to a constant? Since it's always the same? 
2. Make the type of the function also return literal?
3. Write a unit test calling for this function checking against constant being equal to `HistoricalPriceOracle.MANUAL.serialize_for_db()` so that if that ever changes we catch it.